src/Assets/Asset.php

```diff
@@ -1209,9 +1334,49 @@ public function print_before( $thing ) {
 	 * Enqueue the asset.
 	 *
 	 * @since 1.0.0
```

Comment:
```suggestion
	 * @since 1.0.0
	 * @since 1.3.1 Returns itself to enable chaining.
```

Reply:
Whoops, I accepted the default commit message and that's horrible. Hahaha